### PR TITLE
[Eigen] Add alternative site assocation path.

### PR DESCRIPTION
**Pull request by alloy**
I _think_ this is correct, as per [the Express docs](http://expressjs.com/en/api.html#app.mountpath), but this app isn’t really easily testable locally, so please review properly.

### CODE REVIEW

**Comment by broskoski** [comment]
I _always_ review properly

File: server.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ var path = require('path');
 var fs = require('fs');
 var app = express();
 app.use(morgan('combined'));
-app.use('/apple-app-site-association', artsyEigenWebAssociation);
+app.use(['/.well-known/apple-app-site-association', '/apple-app-site-association'], artsyEigenWebAssociation);
```
Comment (broskoski) [inline server.js]: This can be `/(.well-known/)?apple-app-site-association` instead of an array.

**Comment by alloy** [comment]
Clearly you do, see 🙇

**Comment by broskoski** [comment]
👍

**Pull request merged**